src/pages/history/history.jsx

```diff
@@ -0,0 +1,30 @@
+import { HistoryCard } from '../../components/historyCard/historyCard';
+import { Navbar } from '../../components/navbar/navbar';
```

Comment:
you can create index.js file for all components to import @once like import { HistoryCard,Navbar } from '../../components/index'

Reply:
thank you, i will implement that